sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosClient.java

```diff
@@ -136,7 +147,7 @@ TokenResolver getTokenResolver() {
 
     /**
      * CREATE a Database if it does not already exist on the service
-     * The {@link Mono} upon successful completion will contain a single cosmos database response with the 
+     * The {@link Mono} upon successful completion will contain a single cosmos database response with the
```

Comment:
Would this read better with the following:

`Upon successful complete, a {@link Mono} containing the cosmos database response with the created or existing database will be returned`

Potentially remove the statement given that its similar to the `@return` statement. Also, should cosmos be capitalized given it is the service's name?